bindings/python/test/trajectory/test_orbit.py

```diff
@@ -98,6 +98,10 @@ def test_get_pass_with_revolution_number(self, orbit: Orbit):
         assert isinstance(pass_, Pass)
         assert pass_.is_defined()
 
+        assert (
+            orbit.get_pass_with_revolution_number(0, Duration.minutes(10.0)) is not None
```

Comment:
```suggestion
            orbit.get_pass_with_revolution_number(2, Duration.minutes(10.0)) is not None
```
This test is failing cause you can't compute pass numbers in the past

Reply:
Awesome great catch. Thanks man!